src/main/java/no/unit/nva/cristin/projects/NvaProjectBuilder.java

```diff
@@ -46,19 +44,28 @@
     public static NvaProject mapCristinProjectToNvaProject(CristinProject cristinProject) {
         NvaProject nvaProject = new NvaProject();
 
-        nvaProject.setContext(PROJECT_CONTEXT_URL);
-        nvaProject.setId(buildUri(NVA_PROJECT_BASE_URL, cristinProject.cristinProjectId));
+        // TODO: NP-2384: Remember to use setContext when serializing only a single NvaProject
+        nvaProject.setId(buildUri(Constants.BASE_URL, cristinProject.cristinProjectId));
         nvaProject.setType(PROJECT_TYPE);
         nvaProject.setIdentifier(
             Collections.singletonList(Map.of(TYPE, CRISTIN_IDENTIFIER_TYPE, VALUE, cristinProject.cristinProjectId)));
-        nvaProject.setTitle(cristinProject.title.get(cristinProject.mainLanguage));
+        if (cristinProject.title != null) {
+            nvaProject.setTitle(cristinProject.title.get(cristinProject.mainLanguage));
+            nvaProject.setAlternativeTitles(extractAlternativeTitles(cristinProject));
+        }
         nvaProject.setLanguage(buildUri(TEMPORARY_LANGUAGE_URL));
-        nvaProject.setAlternativeTitles(extractAlternativeTitles(cristinProject));
         nvaProject.setStartDate(cristinProject.startDate);
         nvaProject.setEndDate(cristinProject.endDate);
-        nvaProject.setCoordinatingInstitution(mapCristinInstitutionToNvaOrganization(
-            cristinProject.coordinatingInstitution.institution));
-        nvaProject.setContributors(transformCristinPersonsToNvaContributors(cristinProject.participants));
+        if (cristinProject.coordinatingInstitution != null) {
+            nvaProject.setCoordinatingInstitution(attempt(() ->
+                mapCristinInstitutionToNvaOrganization(cristinProject.coordinatingInstitution.institution))
+                .orElse(failure -> null));
+        }
+        if (cristinProject.participants != null) {
+            nvaProject.setContributors(attempt(() ->
+                transformCristinPersonsToNvaContributors(cristinProject.participants))
+                .orElse(failure -> null));
+        }
```

Comment:
Extract method and put the if inside the extracted method so that it will become:
nvaProject.setContributors(extractContriibutors(cristinProject))